demo.py

```diff
@@ -49,14 +48,8 @@ def drive(track, offset):
             assert len(status) == 794, len(status)
             absPosX, absPosY, absPosZ = struct.unpack_from('fff', status, 44)
             angX, angY, angZ = struct.unpack_from('fff', status, 56)
-            num_cars = struct.unpack_from('B', status, 128)
-            posX = struct.unpack_from('f', status, 168)[0]
-            posY = struct.unpack_from('f', status, 324)[0]
-            speed = struct.unpack_from('f', status, 480)[0]
```

Comment:
well, you removed a bit more (this code was for the opponents position ... and it is probably good idea to remove it from demo.py ... but I would double check that these coordinates are also with offset(0,0) now [you can test it via modification of ver0.py to another port say, 3002] - we can do this later.